src/app/learn/reference/configuration/index.html

```diff
@@ -167,6 +167,11 @@
     The <sky-code>host</sky-code> configuration option specifies configuration options for communication with SKY UX Host.
   </p>
   <ul>
+    <li>
+      <p>
+        <sky-code>frameOptions</sky-code> — Specifies an <sky-code>X-Frame-Options</sky-code> value for SKY UX Host HTTP headers to set security preferences for the SPA's embedded iframes. This property is for internal Blackbaud use only.
```

Comment:
I believe we will want to be more generic than this.

[Proposed description for same changes to config](https://github.com/blackbaud/skyux-config/pull/34/files#diff-f93307ffacdcee9c3a3463d83955d866R176)